<?xml version="1.0"?>

<launch>
  <machine
    name="PI_3"
    address="192.168.1.11"
    user="peter"
    password="atalante"
    env-loader="/home/peter/catkin_ws/src/remoteenv.sh"
  />
  
  <!-- AECU Unidriver -->
  <node
    machine="PI_3"
    name="aecu_unidriver"
    pkg="unification_roscontrol"
    type="aecu_unidriver.py"
  />
</launch>